backend/src/main/java/com/woowacourse/f12/exception/badrequest/DuplicatedCategoryProfileProductException.java

```diff
@@ -0,0 +1,10 @@
+package com.woowacourse.f12.exception.badrequest;
+
+import static com.woowacourse.f12.exception.ErrorCode.INVALID_REQUEST_BODY;
+
+public class DuplicatedCategoryProfileProductException extends InvalidValueException {
+
+    public DuplicatedCategoryProfileProductException() {
+        super("대표장비는 카테고리가 중복될 수 없습니다.", INVALID_REQUEST_BODY);
+    }
```

Comment:
```suggestion
    public DuplicatedProfileProductCategoryException() {
        super("대표장비는 카테고리가 중복될 수 없습니다.", INVALID_REQUEST_BODY);
    }
```

단어 순서가 이쪽이 더 좋을 것 같네요